benchmarks/Support/Benchmark.php

```diff
@@ -243,16 +247,20 @@ protected function createPredis(): Predis
      */
     public function getBenchmarkMethods(string $filter): array
     {
+        $exclude = null;
+        if (! extension_loaded('redis')) {
+            $exclude = 'PhpRedis';
```

Comment:
We need to show a warning message when PhpRedis was excluded.

Reply:
done